src/libraries/System.Private.Xml/src/System/Xml/Serialization/XmlSerializer.cs

```diff
@@ -242,7 +243,7 @@ public XmlSerializer(Type type, string defaultNamespace)
             }
         }
 
-        public XmlSerializer(Type type, XmlAttributeOverrides overrides, Type[] extraTypes, XmlRootAttribute root, string defaultNamespace, string location)
+        public XmlSerializer(Type type, XmlAttributeOverrides? overrides, Type[]? extraTypes, XmlRootAttribute? root, string? defaultNamespace, string? location)
```

Comment:
Many overloads that funnel into this one should be able to receive null values but are not annotated as such.
See lines 159, 164, 168 & 172.